Source/Core/Core/NetPlayCommon.cpp

```diff
@@ -0,0 +1,228 @@
+// Copyright 2021 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Core/NetPlayCommon.h"
+
+#include <lzo/lzo1x.h>
+
+#include "Common/FileUtil.h"
+#include "Common/IOFile.h"
+#include "Common/MsgHandler.h"
+#include "Common/SFMLHelper.h"
+
+namespace NetPlay
+{
+static constexpr u32 LZO_IN_LEN = 1024 * 64;
```

Comment:
constexpr variables at namespace scope already have internal linkage by default, so `static` is not required here.